migrations/2023-02-07-030958_community-collections/up.sql

```diff
@@ -0,0 +1,2 @@
+alter table community add column moderators_url text unique;
```

Comment:
Before doing these, always check what the other similar columns look like.

They are all:

`followers_url              | character varying(255)      |           | not null | generate_unique_changeme()`

Reply:
Changed the type to varchar(255). The column is intentionally nullable because its optional, so default value is not necessary.